ql/src/test/queries/clientpositive/redshift_data_connector.q

```diff
@@ -0,0 +1,65 @@
+-- SORT_QUERY_RESULTS
+SHOW CONNECTORS;
+
+-- CREATE with comment
+CREATE CONNECTOR redshift_test
+TYPE 'redshift'
+URL 'jdbc:redshift://redshift-cluster-1.c1gffkxfot1v.us-east-2.redshift.amazonaws.com:5439/dev'
```

Comment:
this seems like an external resource - that's not something we should have in a unit test

Reply:
Agree with Zoltan, I don't think we can add this. Very likely now the Redhshift instance specified here may not be available and the test will fail.

Since Postgres and Redshift have many similarities I would suggest adding at least CONNECTOR test with Postgres. An easy way to do it would be using `--!qt:database:postgres:` option.

Reply:
As mentioned previously, the test should have `SELECT` and `EXPLAIN SELECT` queries to verify the type mapping is done correctly.